llvm/test/Transforms/SimplifyCFG/X86/hoist-loads-stores-with-cf.ll

```diff
@@ -759,6 +757,43 @@ if.true:
   ret i32 %res
 }
 
+define i32 @multi_successors(i1 %c1, i32 %c2, ptr %p) {
```

Comment:
Is this a negative test? If so, the function name should start with "not", and need some comments. e.g. "not_maximum_alignment" in this file.

Reply:
Done.